README.md

```diff
@@ -1,38 +1,23 @@
-# create-svelte
+# odict-web
 
-Everything you need to build a Svelte project, powered by [`create-svelte`](https://github.com/sveltejs/kit/tree/master/packages/create-svelte);
+odict-web is an experimental web frontend for browsing ODict dictionaries.
 
-## Creating a project
+---
 
-If you're seeing this, you've probably already done this step. Congrats!
+## Developer Setup
 
-```bash
-# create a new project in the current directory
-npm init svelte@next
+Build your `node_modules`:
 
-# create a new project in my-app
-npm init svelte@next my-app
 ```
-
-> Note: the `@next` is temporary
-
-## Developing
-
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
-
-```bash
-npm run dev
-
-# or start the server and open the app in a new browser tab
-npm run dev -- --open
+yarn install
 ```
 
-## Building
-
-Before creating a production version of your app, install an [adapter](https://kit.svelte.dev/docs#adapters) for your target environment. Then:
+Run the project:
 
-```bash
-npm run build
 ```
+yarn dev
```

Comment:
Maybe outside of the scope of this PR (overall LG! we should merge), but I think we should set up a docker-compose file that can spin up a Postgres image, then run our seeder to dump our base dictionaries into it to make dev a bit more seamless.

Also, after this merges, I'm curious how you'd like to coordinate building out the FE. I have some half-assed designs I did in Figma a long time ago but I think they need some revisiting 😅

Reply:
I'll merge this in now! 

The docker-compose file sounds like a great idea. Then developers could easily jump right into the code base! Lol for the frontend I'd be down to just share a Figma (Adobe now 😢) sketch file and do some wireframes of some sort.